qa/tasks/mgr_thrash.py

```diff
@@ -0,0 +1,144 @@
+"""
```

Comment:
This need to be run in the context of manager plugins not fs:thrash or fs:workloads

Reply:
OK - so running with fs:thrash sounds fine to me. @batrick thoughts on this?